web_hideleftmenu/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Web Hide Left Menu
+==============
+
+This module adds a menu item after logout link to hide or show left menu
+
+It is usefull on small screens or tree views with many columns
+
+Installation
+============
+Just install module
+
+Configuration
+=============
+
+No configuration
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Click on new menu item "Hide/Show Menu"on dropdown menu after logout menu item.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
+.. branch is "8.0" for example
+
+Known issues / Roadmap
+======================
+
+* ...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{project_repo}/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
+{project_repo}/issues/new?body=module:%20
+{module_name}%0Aversion:%20
+{branch}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+
+Images
+------
+
+
+Contributors
+------------
+
+* Vauxoo
```

Comment:
any email?
as reminder, if this is a company it should in Author section. If individual, put it in this contributor section.